src/components/UserUpdateForm/UserUpdateForm.tsx

```
@@ -0,0 +1,133 @@
+          // 회원 가입 정보 입력 시
+          case "/join":
+            const response = await axios.post(
+              `${process.env.NEXT_PUBLIC_API_BASE_URL}/users`,
+              user,
+              { withCredentials: true }
+            );
+            const redirectUrl = response.data;
+            router.push(redirectUrl);
+          // 회원 정보 수정 시
+          case "/mypage/edit":
+            await axios.put(
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)